frontend/src/components/common/form/StepForm/SteppedForm.md

```diff
@@ -0,0 +1,32 @@
+SteppedForm example:
+
+```js
+import { useFormStepper } from './context';
+
+const FormContent = () => {
+  const stepper = useFormStepper();
+  return (
+    <>
+      <input name="data.name" />
+      <p>Form inputs and control button here</p>
+    </>
+  );
+};
+
+<SteppedForm
```

Comment:
Does this get styled correctly as it is not javascript?

Reply:
This is component read me using markdown syntax